src/README.md

```diff
@@ -9,3 +9,53 @@ Key features include:
 - **Native integration with Pyroscope**: Seamlessly integrates with Pyroscope backend to provide detailed performance profiling and analysis.
 - **Query-Less Browsing**: Navigate profiling data without the need for complex queries.
 - **AI-Powered flame graph analysis**: uses a large-language model (LLM) to assist with flame graph data interpretation so you can identify bottlenecks, and get to the bottom of performance issues faster.
+
+> [!IMPORTANT]
+> Explore Traces is presently undergoing active development and is offered in a preview state. Subsequent updates are likely to incorporate significant changes that may impact existing functionality.
+
+![Exlore Profiles main screen](/media/docs/explore-profiles/explore-profiles-homescreen.png)
+
+## Before you begin
+
+To use Explore Profiles with Grafana Cloud, you need:
+
+- A Grafana Cloud account
+- A Grafana stack in Grafana Cloud with a configured [Pyroscope data source](https://grafana.com/docs/grafana-cloud/connect-externally-hosted/data-sources/pyroscope/) receiving profiling data
+
+To use Explore Traces with self-hosted Grafana open source or Grafana Enterprise, you need:
+
+- Your own Grafana instance running 11.0 or newer
+- Pyroscope 1.7 or newer
+- A configured [Pyroscope data source](https://grafana.com/docs/grafana/latest/datasources/pyroscope/) receiving profiling data
+
+## Getting started
+
+Refer to the [Explore Profiles](https://grafana.com/docs/grafana-cloud/visualizations/simplified-exploration/profiles/) documentation.
+For instructions installing, refer to the [access and installation instructions](https://grafana.com/docs/grafana-cloud/visualizations/simplified-exploration/profiles/).
+
+## Resources
+
+- [Documentation](https://grafana.com/docs/grafana-cloud/visualizations/simplified-exploration/profiles/)
+- [CHANGELOG](https://github.com/grafana/explore-profiles/releases)
+- [GITHUB](https://github.com/grafana/explore-profiles/)
+
+## Contributing
```

Comment:
```suggestion
a [pull request](https://github.com/grafana/explore-profiles/pull/new).
```


I think here we should link to the "new PR" web page instead of explaining a PR.

Reply:
This does sound like a better link, but maybe we should have this on line 46?

Reply:
I'll make the change below.

Reply:
Whoops yes you're right. I reviewed this from the GitHub app so I apologize for putting this suggestion in the wrong place.